getting-started-service-registry/README.adoc

```diff
@@ -0,0 +1,197 @@
+.  In the {product} web console left menu, go to *Streams for Apache Kafka* > *Service Accounts* and click *Create service account* to generate the credentials that you'll use to connect to a Service Registry instance.
++
+NOTE: This is a temporary location for generating service accounts. In the next release, this location for generating service accounts and the granularity of the permissions will change.
+. Copy the generated *Client ID* and *Client Secret* to a secure location.
++
+IMPORTANT: The generated credentials are displayed only one time, so ensure that you've successfully and securely saved the copied credentials before closing the credentials window.
+
+. After you save the generated credentials to a secure location, select the confirmation check box in the credentials window and close the window.
+. For SASL/OAUTHBEARER authentication method, use `https://identity.api.openshift.com/auth/realms/rhoas/protocol/openid-connect/token` as the *Token endpoint URL* and store it to a secure location. This is the endpoint that you'll use with your service account credentials to authenticate the connection to this Service Registry instance.
+
++
+NOTE: SASL/PLAIN authentication is also available for tools and libraries that don't support SASL/OAUTHBEARER, but SASL/OAUTHBEARER is recommended whenever possible. With SASL/PLAIN authentication, you use only the service account credentials to authenticate the connection to the Kafka instance.
+
++
+You’ll use the service account information that you saved to configure your application to connect to your Service Registry instances when you're ready. For example, if you plan to use https://github.com/edenhill/kafkacat[Kafkacat^] to interact with your Kafka instance and deserialize Avro messages using {product}, you'll use this information to set your Schema Registry URL in the client environment variables.
+
+.Verification
+ifdef::qs[]
+* Did you save the Service Registry instance URL to a secure location?
+* Did you save the client credentials to a secure location?
+endif::[]
+ifndef::qs[]
+. Verify that the Service Registry instance URL is saved to a secure location.
+. Verify that the client credentials are saved to a secure location.
+endif::[]
+
+[id="proc-uploading-schema_{context}"]
+== Uploading a schema to {product}
+
+After you create a Service Registry instance, you can upload a schema and start serializing/deserializing messages with the schema.
+
+.Prerequisites
+* You've created a Service Registry instance and the instance is in *Ready* state.
+
+.Procedure
+. In the *Service Registry Instances* page of the web console, select the name of the Service Registry instance that you want to upload a schema to.
+. Click *Upload Artifact* and fill up the form to define the schema details. Click *Upload* to complete the operation.
```

Comment:
Suggest moving "Click Upload to complete the operation" to a separate step after the description of each field below.